genie_registry/assay.py

```diff
@@ -144,18 +144,20 @@ def _validate(self, assay_info_df, project_id):
                 self.syn, "sample", databaseToSynIdMappingDf=db_to_syn_map_df
             )
             uniq_seq_table = self.syn.tableQuery(
-                f"select distinct(SEQ_ASSAY_ID) as uniq_seq from {sample_synid}"
+                f"select distinct(SEQ_ASSAY_ID) as seq from {sample_synid} "
+                f"where CENTER = '{self.center}'"
             )
             uniq_seq_df = uniq_seq_table.asDataFrame()
             # These are all the SEQ_ASSAY_IDs that are in the clinical database
             # but not in the assay_information file
-            missing_seqs = uniq_seq_df['uniq_seq'][
-                ~uniq_seq_df['uniq_seq'].isin(all_seq_assays)
+            missing_seqs = uniq_seq_df['seq'][
+                ~uniq_seq_df['seq'].isin(all_seq_assays)
```

Comment:
Are the elements of all_seq_assays guaranteed to be an exact match (capitalization, hyphen versus underscore) to values in the sample file?

Reply:
well so it is up to the sites themselves to format their SEQ_ASSAY_IDs correctly.  So if their seq assay ids don't match whats in the clinical file, then it should probably fail.